src/twisted/web/test/test_agent.py

```diff
@@ -62,8 +62,8 @@
     from twisted.internet import ssl as _ssl
 except ImportError:
     ssl = None
-    skipWhenNoSSL = "SSL not present, cannot run SSL tests."
```

Comment:
If we're moving the message text out to the individual tests anyway, I think it'd be clearer to have a single flag here named "sslPresent" or something, and use `@skipIf(not sslPresent, ...)` and `@skipIf(sslPresent, ...)` as appropriate

Reply:
that makes sense to me.  I've changed it to `@skipIf(not sslPresent`